package.json

```
@@ -24,7 +24,7 @@
     "web-vitals": "^0.2.4"
   },
   "scripts": {
-    "start": "export SET NODE_OPTIONS=--openssl-legacy-provider && react-scripts start",
+    "start": "react-scripts start",
```

This won't work on my machine. I think you only remove it if you are satisfying some Node.js version. It's surprising to me that the option  itself (which makes it work on my machine) breaks it for you because it should be a general solution. Could you let me know

- What error do you get when the option is set?
- What Node.js version do you have?

I think this can rather be a separate issue so I can merge the rest of the PR.

```suggestion
    "start": "export SET NODE_OPTIONS=--openssl-legacy-provider && react-scripts start",
```

---

Hmmm. That's why I hate Node. 

My version of node is v12.22.9. My version of npm is 8.5.1. Running without the export just works, adding it back gives me : 

node: --openssl-legacy-provider is not allowed in NODE_OPTIONS

---

The easiest possible fix I can imagine is for you to reject this change, keeping the --openssl version as the canonical one. And adding contributor advice about it so that ubuntu devs don't get burned by it. I can remove this file from the commit and force-push the branch and the PR would update automatically.

Anything other than that would be a deep dive into whatever dependency you have that requires this openssl option. Maybe try Deno. I'm only half-joking.

---

> The easiest possible fix I can imagine is for you to reject this change, keeping the --openssl version as the canonical one. 
Yes, dear. You could have equivalently committed my suggestion.
> And adding contributor advice about it so that ubuntu devs don't get burned by it.
My gut feelings suggest that this is about the Node version but will look into it anyway.